apps/settings/js/settings.js

```diff
@@ -67,7 +67,9 @@ var Settings = {
     //     points to the root panel.
     if (this._currentActivity !== null &&
           (hash === '#home' ||
-          (hash === '#root' && Settings._currentActivitySection !== 'root'))) {
+          (hash === '#root' && Settings._currentActivitySection !== 'root') ||
+          (hash === '#call-iccs' && Settings._currentActivitySection !== 'root')
```

Comment:
should be `!== '#call-iccs'` because the intention was to prevent the activity from being closed when back to `#call-iccs`.